code/modules/mob/living/carbon/carbon.dm

```diff
@@ -771,14 +771,19 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	return(istype(src.wear_mask, /obj/item/clothing/mask/muzzle))
 
 /mob/living/carbon/resist_buckle()
+	var/breakouttime = 600
+
 	spawn(0)
 		resist_muzzle()
 	if(restrained())
+		var/obj/item/I = handcuffed
+		breakouttime = I.breakouttime
```

Comment:
Don't cable ties have a different value than 600?

Reply:
Yeah I just set it to 600 by default to avoid possible complications. The breakout time is adjusted to the item they're 'handcuffed' with, with `breaktouttime = I.breakouttime`.

Reply:
I mean this: https://github.com/ParadiseSS13/Paradise/blob/5936bb98fea2bc46769df7ee6bdf57a5f3f3db37/code/game/objects/items/weapons/handcuffs.dm#L95 will override the 60 seconds that are in the code right now.

Reply:
Yeah that's what the beakouttime variable gets set to. I declare them at 600 just as a default, but then the `var/obj/item/I = handcuffed` gets the item the person is handcuffed with, and then we set the breakoutime to whatever that item's breakout time is, meaning the 600 gets changed to 300 for cable cuffs, 0 for toy cuffs, 450 for zipties, etc. 

if this is about the display time, then that's the same display time the regular handcuff code uses. (This will take 0.5 minutes to break out of, as an example.)

Reply:
In what scenario are you handcuffed with cuffs without a breakout time? ANd even if it did, then breakouttime would just be treated as 0 which isn't the worst issue, imo.

Reply:
That is true. I'll look into cleaning this up a little bit.